.travis.yml

```diff
@@ -29,9 +29,3 @@ script:
 - if [ "$OS" = "linux" ]; then bash "$TRAVIS_BUILD_DIR"/tools/travis/ci-build-linux.sh; fi
 - if [ "$OS" = "windows" ]; then bash "$TRAVIS_BUILD_DIR"/tools/travis/ci-build-windows-ms.sh; fi
 - if [ "$OS" = "linux" -a "$TRAVIS_PULL_REQUEST" != "false" ]; then bash "$TRAVIS_BUILD_DIR"/tools/travis/pr-build.sh; fi
-- cd "$TRAVIS_BUILD_DIR"
-- echo "checking:"
-- find lua_modules lua_examples -iname "*.lua" -print0 | xargs -0 echo
-- find lua_modules lua_examples -iname "*.lua" -print0 | xargs -0 $LUACC -p
```

Comment:
Test compiling the lua sources can please stay in and the luacheck line might only get commented as we tend to want it back at some pint in time.

Reply:
@HHHartmann, Gregor if you feel strongly about this just do the PR and merge it. :smile: